ssl_proxy_vars.yml

```diff
@@ -0,0 +1,24 @@
+---
+nginx_sites:
+  ssl_reverse_proxy:
+    - listen 443 ssl
+    - ssl_certificate wildcard.tsuru.paas.alphagov.co.uk.site.chain.crt
+    - ssl_certificate_key wildcard.tsuru.paas.alphagov.co.uk.key
+    - server_name tsuru.paas.alphagov.co.uk
```

Comment:
We're going to need to do something about setting the server_name correctly to match the certificate name - I should have noticed that in my own SSL cert PR just now.